README.md

```diff
@@ -69,6 +69,7 @@ To install a package into the virtual environment:
 uv pip install flask                # Install Flask.
 uv pip install -r requirements.txt  # Install from a requirements.txt file.
 uv pip install -e .                 # Install the current project in editable mode.
+uv pip install package@.            # Install the current project from disk
```

Comment:
Should we separate so it's easier to read? (honestly not sure) cc @charliermarsh 

```suggestion
uv pip install "package @ ."        # Install the current project from disk
```

Reply:
I'm a little worried about people thinking they should do this instead of `-e .` (more common)

Reply:
I don't have strong opinions about the formatting,  and based on opened issues a I'd suggest that its a common enough pattern that is also non obvious that adding the pattern might save others some frustration.

Reply:
fwiw, this syntax will put in `pyproject.toml` when using `rye add mypkg --path D:/test/mypkg`:
```toml
dependencies = ["mypkg @ file:///D:/test/mypkg"]
```